modules/site/.gitignore

```diff
@@ -25,4 +25,7 @@ testem.log
 /e2e/*.map
 
 #System Files
-.DS_Store
\ No newline at end of file
+.DS_Store
+
+# Meta files generated by ng2-material build
+public/*.json
```

Comment:
IMO, it should be in a hotfix cause it's not directly related to this PR and it concern several pending pull requests.